src/services/user.service.js

```diff
@@ -0,0 +1,14 @@
+import db from '../database/models';
+
+const UserService = {
+  getUserLineManager: async (id) => db.Users.findOne({ where: { id }, attributes: ['lineManager'] }),
```

Comment:
@urbanishimwe  great job but you falling off the truck of  consistency in our functions naming convention